versioned_hdf5/meson.build

```diff
@@ -61,11 +60,21 @@ cython_args = [
 # meson build && pushd build && meson compile && popd && firefox $(find build/ -name "*.html")
 
 py.extension_module(
-    'subchunk_map',
+    'cytools',
```

Comment:
I see the comment below - in the future it might be possible to do this with Meson generators, I think? Anyway, this is fine for now.

Reply:
I'm hesitant to call `ln -s cytools.py cytools.pyx` at build time, as it would break Windows builds, whereas symlink checked in git are a very well known problem for windows users that's easily solved with a stackoverflow search.